.pre-commit-config.yaml

```diff
@@ -1,14 +1,14 @@
 ---
 repos:
-  - repo: git://github.com/Lucas-C/pre-commit-hooks
-    rev: v1.1.9
+  - repo: git://github.com/pre-commit/pre-commit-hooks
```

Comment:
Nit, no need to have this repo imported twice. Just use the `pre-commit/pre-commit-hooks` hook id for `trailing-whitespace` as it is in the master.

Reply:
Yeah, not sure what I was thinking there.